parser/conn.go

```diff
@@ -51,3 +55,22 @@ type (
 		TunnelParents string `bson:"tunnel_parents" bro:"tunnel_parents" brotype:"set[string]"`
 	}
 )
+
+// GetHostName is our method for collecting host name
+// Temporary function
+func (in *Conn) IsWhiteListed(whitelist []string) bool {
```

Comment:
Ben's whitelisted code for import. Each type of log defines its own iswhitelisted func so it can check the whitelist against the appropriate fields